packages/@ourworldindata/grapher/src/stackedCharts/StackedBarChart.tsx

```diff
@@ -674,6 +675,25 @@ export class StackedBarChart
 
     defaultBaseColorScheme = ColorSchemeName.stackedAreaDefault
 
+    /**
+     * Colour positive and negative values differently if there is only one series
+     * (and that series has both positive and negative values)
+     */
+    @computed get shouldUseValueBasedColorScheme(): boolean {
+        return (
+            this.rawSeries.length === 1 &&
+            this.rawSeries[0].rows.some((row) => row.value < 0) &&
+            this.rawSeries[0].rows.some((row) => row.value >= 0)
```

Comment:
I think it makes sense to check for strictly positive values here.
```suggestion
            this.rawSeries[0].rows.some((row) => row.value > 0)
```
That way, we don't also catch the case where we only have negative and zero values (don't think that's gonna be common, but still).